Highlight/languages/dsconfig.json

```diff
@@ -1 +1 @@
-{"keywords":"dsconfig","contains":[{"className":"keyword","begin":"^dsconfig","end":"\\s","excludeEnd":true,"relevance":10},{"className":"built_in","begin":"(list|create|get|set|delete)-(\\w+)","end":"\\s","excludeEnd":true,"illegal":"!@#$%^&*()","relevance":10},{"className":"built_in","begin":"--(\\w+)","end":"\\s","excludeEnd":true},{"className":"string","begin":"\"","end":"\""},{"className":"string","begin":"'","end":"'"},{"className":"string","begin":"[\\w-?]+:\\w+","end":"\\W","relevance":0},{"className":"string","begin":"\\w+-?\\w+","end":"\\W","relevance":0},{"className":"comment","begin":"#","end":"$","contains":[{"begin":"\\b(a|an|the|are|I'm|isn't|don't|doesn't|won't|but|just|should|pretty|simply|enough|gonna|going|wtf|so|such|will|you|your|they|like|more)\\b"},{"className":"doctag","begin":"(?:TODO|FIXME|NOTE|BUG|XXX):","relevance":0}]}]}
+{"keywords":"dsconfig","contains":[{"className":"keyword","begin":"^dsconfig","end":"\\s","excludeEnd":true,"relevance":10},{"className":"built_in","begin":"(list|create|get|set|delete)-(\\w+)","end":"\\s","excludeEnd":true,"illegal":"!@#$%^&*()","relevance":10},{"className":"built_in","begin":"--(\\w+)","end":"\\s","excludeEnd":true},{"className":"string","begin":"\"","end":"\""},{"className":"string","begin":"'","end":"'"},{"className":"string","begin":"[-?\\w]+:\\w+","end":"\\W","relevance":0},{"className":"string","begin":"\\w+-?\\w+","end":"\\W","relevance":0},{"className":"comment","begin":"#","end":"$","contains":[{"begin":"\\b(a|an|the|are|I'm|isn't|don't|doesn't|won't|but|just|should|pretty|simply|enough|gonna|going|wtf|so|such|will|you|your|they|like|more)\\b"},{"className":"doctag","begin":"(?:TODO|FIXME|NOTE|BUG|XXX):","relevance":0}]}]}
```

Comment:
Can you add this [patch to our `get_language_definitions.php` script](https://github.com/scrivo/highlight.php/blob/146d431e0cc09c71965b5ffcc8a889004cec3a2b/tools/get_language_definitions.php#L40-L58)? These JSON files are generated automatically each time we pull definitions from upstream so this change will get lost otherwise.

This should also be a separate commit rather than being lumped in together with the reference bug fix.

Reply:
> The dsconfig change addresses a change in PCRE that renders `[\\w-?]` invalid.

If you've got a link to a changelog for this, please post that link as a comment in the script; it'll help in the future for remembering why a patch exists.